tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "composite": true,
     "module": "commonjs",
+    "lib": ["es2019", "dom"],
```

Comment:
This is something that comes up every once in a while - is this a breaking change?